env/fs_posix.cc

```diff
@@ -238,7 +238,11 @@ class PosixFileSystem : public FileSystem {
     }
     SetFD_CLOEXEC(fd, &options);
 
-    if (options.use_mmap_reads && sizeof(void*) >= 8) {
+    // Remove 64 bit OS checking. Both 64 and 32 bit OSes use mmap to read files
```

Comment:
I don't think we need to explain an old, unnatural state of the code here. That's what release notes are for.

Reply:
Done